packages/dds/tree/src/test/testHelper.ts

```diff
@@ -0,0 +1,173 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// eslint-disable-next-line import/no-internal-modules
+import { createIdCompressor } from "@fluidframework/id-compressor/legacy";
+import { SchemaFactoryAlpha, TreeViewConfiguration } from "../simple-tree/index.js";
+import { TableSchema } from "../tableSchema.js";
+import { DefaultTestSharedTreeKind } from "./utils.js";
+import { AttachState } from "@fluidframework/container-definitions";
+import {
+	MockContainerRuntimeFactory,
+	MockFluidDataStoreRuntime,
+} from "@fluidframework/test-runtime-utils/internal";
+import { CommitKind, type Revertible } from "../core/index.js";
+
+/**
+ * Define a return type for table tree creation.
+ */
+export interface TableTreeDefinition {
+	/**
+	 * The cell class used to define the table tree schema.
+	 */
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	Cell: any;
+	/**
+	 * The column class used to define the table tree schema.
+	 */
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	Column: any;
+	/**
+	 * The row class used to define the table tree schema.
+	 */
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	Row: any;
+	/**
+	 * The table class used to define the table tree schema.
+	 */
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	Table: any;
+	/**
+	 * The table tree instance.
+	 */
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	table: any;
+	/**
+	 * The undo stack for the table tree.
+	 */
+	undoStack: Revertible[];
+	/**
+	 * The redo stack for the table tree.
+	 * */
+	redoStack: Revertible[];
+	/**
+	 * Unsubscribe from the table tree events and dispose of the undo/redo stacks.
+	 */
+	unsubscribe: () => void;
+}
+
+/**
+ * Provides a simple table tree with the given size and cell value.
+ */
+export function createTableTree(tableSize: number, cellValue: string): TableTreeDefinition {
+	const schemaFactory = new SchemaFactoryAlpha("test");
+	class Cell extends schemaFactory.object("table-cell", {
```

Comment:
I would recommend defining all of this schema in some outer scope so that you can reference the types in `TableTreeDefinition`, etc. (will remove the need for using `any` in a bunch of places)

Reply:
It probably also makes sense to export the schema types so that the tests can reference them directly.

Reply:
Move the schema types to outside